### DISCUSSION TITLE
Writing my first smart contract in solidity language

### DISCUSSION THREAD

**Discussion by Joezzyongit** [Cyfrin/foundry-full-course-cu#1474] [General]
Hi fellow blockchain devs! I am writing to encourage everyone taking this journey, we got it and we are the next devs to build the next big thing in the crypyto space keep going and don't stop!!

**Comment by EngrPips** [comment]
Good energy @Joezzyongit , thats the spirit to attain greatness.

**Reply by Joezzyongit** [reply]
Thanks man. @EngrPips let's connect and work together what do you say ?

**Reply by EngrPips** [reply]
That would be awesome, let's see what the future holds

**Comment by Gloryjaw** [comment]
Completed solidity fundamentals, excited for starting foundry tomorrow 😊

**Reply by EngrPips** [reply]
Wow thats pretty fast, you are moving so quick hahahahaha

**Reply by Joezzyongit** [reply]
Hi @Gloryjaw, let's connect and work together what do you say ?

**Reply by Gloryjaw** [reply]
That would be great 😀. Currently on last lesson of foundry fundamentals.

**Reply by Joezzyongit** [reply]
That's cool, kindly give me your telegram username or if you're on twitter  anyone that's preferable with you so i can add you up

**Reply by Gloryjaw** [reply]
I followed you on twitter, my username is : @gloryjaw
